enhancements/helm3/namespace-scoped-helm-repository-crd.md

```diff
@@ -64,14 +64,34 @@ kind: ProjectHelmChartRepository
 metadata:
   name: my-enterprise-chart-repo
 spec:
-  url: https://my.chart-repo.org/stable
-
-  # optional and only needed for UI purposes
-  displayName: myChartRepo
```

Comment:
Instead of deleting can we rename to "name". The meaning is the same: Friendly name to display in the UI

Reply:
Updated.